src/dmd/parse.d

```diff
@@ -290,6 +296,22 @@ final class Parser(AST) : Lexer
         Loc lookingForElse; // location of lonely if looking for an else
     }
 
+    version (UseFileManager)
+    {
+        FileManager *filemanager = null;
+
+        void setFileManager(FileManager *fm)
```

Comment:
Don't use `get`/`set` methods in D. Use properties [1]  instead.

[1] https://dlang.org/spec/property.html